src/main/resources/application.properties

```diff
@@ -88,7 +88,7 @@ jwt.sessionTime=2592000
 pulsar-manager.account=pulsar
 pulsar-manager.password=pulsar
 # If true, the database is used for user management
-user.management.enable=true
+user.management.enable=false
```

Comment:
If this option is set to default to `false`, I think we may have some documents that need to be modified in parallel, so I suggest we keep this option false for now @dockerzhang